src/libs/Navigation/AppNavigator/Navigators/OnboardingModalNavigator.tsx

```diff
@@ -40,10 +42,14 @@ function OnboardingModalNavigator() {
         Navigation.isNavigationReady().then(() => {
             // Need to go back to previous route and then redirect to Concierge,
             // otherwise going back on Concierge will go to onboarding and then redirected to Concierge again
-            Navigation.goBack();
-            Report.navigateToConciergeChat();
+            if (isSmallScreenWidth) {
+                Navigation.setShouldPopAllStateOnUP(true);
+                Navigation.goBack(ROUTES.HOME, true, true);
+            } else {
+                Navigation.goBack();
```

Comment:
```suggestion
                Navigation.goBack();
                Report.navigateToConciergeChat();
```

@daledah Let's keep the existing behavior for a wider screen and navigate to concierge chat. [Reason](https://github.com/Expensify/App/pull/45705#issuecomment-2247554578)